src/main/config/emissary.core.SafeUsageChecker.cfg

```diff
@@ -0,0 +1 @@
+ENABLED = TRUE
```

Comment:
To minimize impact, should we initially default to disabled and override downstream, potentially with a flavored config?